themes/xmpp-providers/layouts/partials/provider-list-items.html

```diff
@@ -80,7 +80,12 @@ <h2 class="accordion-header" id="heading-{{ $provider_id }}">
                                     <i class="fa-solid fa-location-dot px-2 text-muted"></i>
                                 </td>
                                 <td>
-                                    Server location(s):
+                                    Server / Data
+                                    {{ if and .serverLocations (eq (len .serverLocations) 1) }}
```

Comment:
```suggestion
                                    {{ if lt (len .serverLocations) 2 }}
```

Sorry, you were right, `lt 2` covers the "unknown" case, I forgot about this.